<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QGroupBox" name="parametros" >
    <property name="geometry" >
     <rect>
      <x>570</x>
      <y>50</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title" >
     <string>Par√°metros</string>
    </property>
    <widget class="QComboBox" name="listaParametros" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="renderHints" >
     <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
    </property>
    <property name="dragMode" >
     <enum>QGraphicsView::RubberBandDrag</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="metodos" >
    <property name="geometry" >
     <rect>
      <x>580</x>
      <y>160</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title" >
     <string>Metodos</string>
    </property>
    <widget class="QPushButton" name="euler" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Euler</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rungekutta" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Runge-Kutta</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ode23" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>ODE_23</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ode45" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>150</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>ODE_45</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu" >
    <property name="title" >
     <string>Menu</string>
    </property>
    <addaction name="actionParametros" />
    <addaction name="actionCondiciones" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuMenu" />
   <addaction name="menuHelp" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionParametros" >
   <property name="text" >
    <string>Parametros</string>
   </property>
  </action>
  <action name="actionCondiciones" >
   <property name="text" >
    <string>Condiciones</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
